overview.md

```diff
@@ -4,81 +4,86 @@ css_id: overview
+   itself. For example, Firefox updates itself through its own application
+   updater.
+
+* **Library package managers** offered by many
+   programming languages for installing additional libraries. Examples include Python's pip/easy_install + PyPI, Perl's CPAN,
+   Ruby's RubyGems, and PHP's Composer.
 
-The problem with this view is that it is only straightforward when there are no
-malicious parties involved throughout the update procedure. If an attacker is
-trying to interfere with these seemingly simple steps, there is plenty that
-they can do.
+* **System package managers** used by operating systems to update and
+   install software on a client system. Examples include Debian's APT,
+   Red Hat's YUM and openSUSE's YaST are examples of these.
 
-TUF is designed to perform the first two steps of the above update procedure,
-while guarding against the majority of attacks that malicious actors have at
-their disposal; especially those attacks that are overlooked by
-security-conscious developers.
+While these systems may vary in how they work, most follow a similar update
+procedure. Obtaining and installing an update simply means:
 
-### Background ###
+* Knowing when an update exists.
+* Downloading the update.
+* Applying the changes introduced by the update.
 
-Let's assume you take the approach that most systems do (at least, the ones
-that even try to be secure). You download both the file you want and a
-cryptographic signature of the file. You already know which key you trust to
-make the signature. You check that the signature is correct and was made by
-this trusted key. All seems well, right? Wrong. You are still at risk in many
-ways, including:
+TUF is designed to perform the first two steps of this procedure,
+while guarding against the majority of attacks that can occur during or
+after the update.
+These include threats that other software security strategies may not take into
+account, such as when:
 
 * An attacker keeps giving you the same file, so you never realize
   there is an update.
 
 * An attacker gives you an older, insecure version of a file that you
-  already have, so you download that one and blindly use it thinking it's
+  already have, so you download it and blindly use it thinking it's
   newer.
 
-* An attacker gives you a newer version of a file you have but it's not
+* An attacker gives you a newer version of a file you have but it's still not
   the newest one. It's newer to you, but it may be insecure and
   exploitable by the attacker.
 
 * An attacker compromises the key used to sign these files and now you
-  download a malicious file that is properly signed.
+  download a file that is still malicious file, even if it is properly signed.  
 
-These are just some of the attacks software update systems are vulnerable to
-when only using signed files. See [Security](/security.md) for a full list of
+The [Security](/security.md) section offers a full list of the
```

Comment:
correction: "The Security page offers ..."

Reply:
Check. Fixed.